### ISSUE TITLE
Quest Search: Filter or sort by number or reviews

### ISSUE THREAD

**Issue by joecot** [ExpeditionRPG/expedition#773]
Either:
* the ability to filter based on number of reviews (only show quests with 5/10/etc reviews)
* the ability to sort based on number of reviews (as opposed to sort by rating)
* or both.

Sorting by highest rated is great and all, but often the highest rated ones only have a couple of reviews, which isn't a great indicator of whether it's a really good quest. For quests I'm running for other people, I'd like to be able to quickly see quests that have been played often and still have a high rating.

**Comment by toddmedema** [comment]
Good point! I've updated this to be "most played" / "most popular" since I've seen that as a common filter option in other places

**Comment by toddmedema** [comment]
@joecot Another option here: What if instead of creating a different filter, we improved how the default filter works? I.e. did a better job of prioritizing quests that are highly rated AND have a lot of ratings? Right now we consider that any quest with at least 5 ratings "has a lot of ratings", but we could bump up that threshold to be more like 15 ratings
